### PR TITLE
Remove one-item exception from the focus visible rule

### PR DESCRIPTION
**Pull request by WilcoFiers**
- AGWG recently discussed and agree to remove the exception of pages with a single control not needing a focus indicator. (https://github.com/w3c/wcag/pull/2653/files) This rule has been aligned with that decision
- Update WCAG 2.2 reference, as those have changed

Need for Call for Review: 1 week, since the applicability changed

---

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by tbostic32 — APPROVED** [review]

**Review by Jym77 — DISMISSED** [review]
Great simplification 🎉 
We need to remove all the "dummy buttons" from the examples.

**Review by Jym77 — APPROVED** [review]

File: _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md [hunk 1]
```diff
@@ -22,62 +22,60 @@ acknowledgments:
...
+The rule applies to any element which is part of [sequential focus navigation][] in an [HTML document](https://html.spec.whatwg.org/#document).
 
 ## Expectation
 
 For each target element, there is at least one device pixel inside the [scrolling area][] of the [viewport][] whose [HSL color value](https://www.w3.org/TR/css-color-3/#hsl-color) is different when the element is [focused][] from when it is not.
 
 ## Assumptions
 
-This rule assumes that documents with only one focusable element automatically satisfy [Success Criterion 2.4.7 Focus Visible][sc247], as stated in [Understanding Success Criterion 2.4.7: Focus Visible][usc247]: "if there is only one keyboard actionable control on the screen, the success criterion would be met".
+_There are currently no assumptions._
```
Comment (HelenBurge) [inline _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md]: Should the _ be removed?

**Review by HelenBurge — DISMISSED** [review]

File: _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md (hunk 1 above)
Comment (daniel-montalvo) [reply _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md]: I wouldn't touch that in this PR, I am taking care aof all this in #1945

Comment (WilcoFiers) [inline _rules/sequentially-focusable-element-has-visible-focus-oj04fd.md]:
```suggestion
There are no assumptions.
```

**Review by HelenBurge — APPROVED** [review]

**Review by tbostic32 — APPROVED** [review]

**Comment by giacomo-petri** [comment]
I believe we should also remove inapplicable example n.2, where we have only one link in page.

**Comment by WilcoFiers** [comment]
@giacomo-petri You are right, thanks! I made the change.

**Pull request merged**